### PR TITLE
Sass support

### PR DESCRIPTION
**Pull request by olofenglund96**
This PR adds support for sass. With it comes a compiler which compiles all files when saving. Default place to safe css is now as a partial in _www/scss/partials. Just don't forget to include the document in index.scss.

### CODE REVIEW

**Comment by olofenglund96** [comment]
I would by the way love some testing of this on different OS!

**Comment by navro** [comment]
This looks very nice! 😄
I will try it on Arch, but someone else will have to test it on winDOS 😉 
Please squash your commits.

**Comment by navro** [comment]
You will also have to fix the deploy script, but you can do that in another PR if you wish

**Comment by navro** [comment]
i have tested this now and it works fine on arch :)

File: deploy/master [hunk 1]
```diff
@@ -20,6 +22,8 @@ git reset --hard origin/master
 sed -i '/<link rel="stylesheet"/s/material/ios/' ./www/index.html
 sed -i '/<platform name="android">/,/<\/platform>/d' ./config.xml
 
+gulp 'scss'
```
[the remark this answers is not in the document]
Comment (navro) [reply deploy/master]: I think so!

File: deploy/production [hunk 2]
```diff
@@ -17,6 +17,11 @@ sed -i 's/509736475453/178540363026/' ./package.json
 sed -i 's/509736475453/178540363026/' ./www/js/push.js
 sed -i 's/stage.fsektionen.se/fsektionen.se/' ./www/js/index.js
 
+gulp 'scss'
+rm -rf 'www/scss'
+rm 'start.js'
+rm 'gulpfile.js'
+
```
[the remark this answers is not in the document]
Comment (navro) [reply deploy/production]: Yes! No need to include those files, it will only make the binaries larger :)

**Comment by FredrikLastow** [comment]
@olofenglund96 Should we close this?

**Comment by olofenglund96** [comment]
This is replaced by #166

**Pull request closed without merging**